### PR TITLE
[Fix-5055][UI] Fix can not upload file to correct dir

### PR DESCRIPTION
**Pull request by lijufeng2016**
## Purpose of the pull request

Fix [Bug][UI] Can not upload file to correct dir #5055

### CODE REVIEW

File: dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue [hunk 1]
```diff
@@ -174,8 +174,8 @@
           formData.append('file', this.file)
           formData.append('type', this.type)
           formData.append('name', this.name)
-          formData.append('pid', this.pid)
-          formData.append('currentDir', this.currentDir)
+          formData.append('pid', localStore.getItem('pid') || this.id)
+          formData.append('currentDir', localStore.getItem('currentDir') || this.currentDir)
```
Comment (chengshiwen) [inline dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue]: This problem may also appear in `definitionUpdate.vue`, `fileChildReUpdate.vue`, `fileReUpload.vue`, `fileUpdate.vue`, `resourceChildUpdate.vue` and `udfUpdate.vue` under directory `dolphinscheduler-ui/src/js/module/components/fileUpdate`, please check together, thanks a lot!

**Review by chengshiwen — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue (hunk 1 above)
Comment (lijufeng2016) [reply dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue]: ok,I will check it

Comment (lijufeng2016) [reply dolphinscheduler-ui/src/js/module/components/fileUpdate/fileChildUpdate.vue]: I fix all bugs for upload resource file and udf,by the way,I fix when delete udf resource file,the page not refresh

**Comment by CalvinKirs** [comment]
@chengshiwen PTAL,Thx

**Comment by chengshiwen** [comment]
> @chengshiwen PTAL,Thx

I will give feedback later

**Comment by zhuangchong** [comment]
+1

**Review by chengshiwen — APPROVED** [review]
+1

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**